packages/atomic/src/components/search/facets/atomic-automatic-facet-generator/atomic-automatic-facet-generator.tsx

```diff
@@ -60,9 +60,12 @@ export class AtomicAutomaticFacetGenerator implements InitializableComponent {
    * @beta - This prop is part of the automatic facets feature.
    * Automatic facets are currently in beta testing and should be available soon.
    *
-   * Specifies whether the automatic facets are collapsed.
+   * Using the value `0` collapses all facets.
+   * Using the value `-1` disables the feature and keeps all facets expanded. Useful when you want to set the collapse state for each facet individually.
```

Comment:
I don't really like the pattern of using `-1` as meaning "disabled", although I realize that `undefined` means "default".

Maybe we could take a string instead? I think Stencil may support union types. For example:
```ts
  @Prop() public collapseFacetsAfter?: number | 'never-collapse';
```

Reply:
It's the same pattern that is already in use in the project for the same kind of property (atomic-facet-manager).

Not using the same pattern would be more problematic.